be1-go/message/messagedata/vote_cast_vote.go

```diff
@@ -15,9 +20,38 @@ type VoteCastVote struct {
 
 // Vote defines a vote of a cast vote
 type Vote struct {
-	ID       string `json:"id"`
-	Question string `json:"question"`
-	Vote     []int  `json:"vote"`
+	ID       string
+	Question string
+	Vote     interface{}
+}
+
+type tempVote struct {
+	ID       string      `json:"id"`
+	Question string      `json:"question"`
+	Vote     interface{} `json:"vote"`
+}
```

Comment:
We need both because I changed the unmarshalling function and it without the tempVote it would go into an infinite loop. I should probably add a comment

Reply:
Ok, yes it may be a good idea to comment it.

Reply:
Also, why you don't have the `json: ??? ` for the all Vote fields ? 😅 
Finally,  maybe also write "TempVote" to follow the codebase naming convention.

Reply:
oh ok didn't know, I understand now.
I think you should really comment it as it differs from other parts of the code. Otherwise looks good 👍